calcite-adapter/src/test/java/software/amazon/documentdb/jdbc/query/DocumentDbQueryMappingServiceBasicTest.java

```diff
@@ -1633,20 +1633,14 @@ void testQueryAndWithTypes() throws SQLException {
         Assertions.assertEquals(
                 BsonDocument.parse(
                         "{\"$project\": {"
-                                + "\"EXPR$0\": {\"$cond\": [{\"$and\": [{\"$eq\": [true, {\"$cond\": ["
-                                + "{\"$and\": [{\"$gt\": [\"$field\", null]}, "
-                                + "{\"$gt\": [{\"$date\": \"2021-01-01T00:00:00Z\"}, null]}]}, "
-                                + "{\"$gt\": [\"$field\", {\"$date\": \"2021-01-01T00:00:00Z\"}]}, null]}]}, "
-                                + "{\"$eq\": [true, {\"$cond\": [{\"$and\": [{\"$gt\": [\"$field\", null]}, "
-                                + "{\"$gt\": [{\"$date\": \"2020-02-01T00:00:00Z\"}, null]}]}, "
-                                + "{\"$lt\": [\"$field\", {\"$date\": \"2020-02-01T00:00:00Z\"}]}, null]}]}]}, true, "
-                                + "{\"$cond\": [{\"$or\": [{\"$eq\": [false, "
-                                + "{\"$cond\": [{\"$and\": [{\"$gt\": [\"$field\", null]}, "
-                                + "{\"$gt\": [{\"$date\": \"2021-01-01T00:00:00Z\"}, null]}]}, "
-                                + "{\"$gt\": [\"$field\", {\"$date\": \"2021-01-01T00:00:00Z\"}]}, null]}]}, "
-                                + "{\"$eq\": [false, {\"$cond\": [{\"$and\": [{\"$gt\": [\"$field\", null]}, "
-                                + "{\"$gt\": [{\"$date\": \"2020-02-01T00:00:00Z\"}, null]}]}, "
-                                + "{\"$lt\": [\"$field\", {\"$date\": \"2020-02-01T00:00:00Z\"}]}, null]}]}]}, false, null]}]}"
+                                + "\"EXPR$0\": {\"$cond\": [{\"$and\": [{\"$eq\": [true, null]}, "
```

Comment:
Looks like this got simplified to `EXPR$0=[AND(null, IS NULL(field))` since the condition can never be true. This actually seems correct then but we don't have a test that checks this yet. May need to add a test that checks that the query returns NULL when field is NULL and FALSE when it is not null.

Reply:
Added test in DocumentDbStatementDateTimeTest.java.